### DISCUSSION TITLE
[Feature] Duplicate Detector: Show Album Name

### DISCUSSION THREAD

**Discussion by chk86** [immich-app/immich#10206] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In addition to whether or not a photo is in an album, it also would be helpful to have the name of the album listed either in the JSON tooltip or in the viewer itself. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by chrisscotland** [comment]
I agree and also can I refer to my related suggestion #10262 which would hopefully address this issue as well.

**Reply by chk86** [reply]
@chrisscotland, your suggestion would resolve my issue! That's my main concern—being able to keep photos in the duplicate detector in existing albums with minimal effort.

**Comment by nielstron** [comment]
In addition to showing Album/Not Album I think it would be great to see the file path or last path component (for external files), since this often substitutes Albums in external libraries.

**Comment by danieldietzler** [comment]
Addressed in #29080
